src/shogun/mathematics/linalg/ratapprox/logdet/LogDetEstimator.cpp

```diff
@@ -4,7 +4,6 @@
  * Authors: Sunil Mahendrakar, Heiko Strathmann, Soumyajit De, Björn Esser
  */
 #include <shogun/base/Parallel.h>
-#include <shogun/base/progress.h>
```

Comment:
why?

Reply:
It was not being used anywhere... we could put it back in when we add progress bar to it (after this gets merged )